src/components/NegotiationList.vue

```diff
@@ -0,0 +1,217 @@
+<template>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+  <header>
+    <h3>
+      List of your Negotiations:
+    </h3>
+  </header>
+    <div class="float-child">
+    <table id="tableComponent" class="table table-hover styled-table">
+        <thead>
+        <tr>
+            <th  v-for="field in headers" :key='field' >
+                {{field}}
+            </th>
+            <th/>
+        </tr>
+        </thead>
+        <tbody v-for="item in negotiations" :key='item'>
+        <tr class="active-row">
+            <td>{{ item.id }}</td>
+            <td>{{ item.payload.project.title }}</td>
+            <td>{{ item.status }}</td>
+            <td>
+                <span style="white-space: pre">
+                    <button type="button" class="editButton" @click="interactModal(item)">
+                        <i class="fa fa-pencil"></i>
+                    Interact
+                </button>
+                <button class="abandonButton" @click="abandonRequest">
+                    <i class="fa fa-trash"></i>
+                    Abandon
+                </button>
+                </span>
+            </td>
+        </tr>
+        </tbody>
+    </table>
+    </div>
+    <div class="float-child">
```

Comment:
Probably it would be more correct to arrange filter table and negotiation table using Bootstrap Layout.

https://getbootstrap.com/docs/5.0/layout/containers/

With this custom panel, the layout is not responsive. If you try to decrease the screen size, the filter table goes down